Foundations/Components/ProfileCircle.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  ProfileCircle.swift
+//  Foundations
+//
+//  Created by José Danilo Silva do Carmo on 23/03/23.
+//
+
+import SwiftUI
+
+struct ProfileCircle: View {
+    @State private var imagePath: String;
+    @State private var profileName: String;
+    
+    init(imagePath: String, profileName: String) {
+        self.imagePath = imagePath;
+        self.profileName = profileName;
+    }
+    
+    var body: some View {
+        VStack {
+            ZStack {
+                Image(imagePath)
+                Circle()
+                    .inset(by: 0)
+                    .stroke(Color("light-yellow"),
+                            lineWidth: 8)
+            }.frame(width: 100, height: 120)
```

Comment:
It's a image with a circle (border) above, limited by 100x120 (to give a false margin in height) because if you don't set a frame size the inset will be a little weird